apps/condo/domains/common/utils/codegeneration/generate.server.utils.js

```diff
@@ -228,6 +228,14 @@ function generateServerUtils (gql) {
         })
     }
 
+    async function softDelete (context, id, extraAttrs = {}) {
+        const attrs = {
+            deletedAt: 'true',
```

Comment:
Please use current time!

Reply:
Looks like this is not necessary, because of this: https://github.com/open-condo-software/condo/blob/1d1819fea3417b861c404976dd404b0a1752e0fb/packages/%40core.keystone/plugins/softDeleted.js#L52